iso15118/secc/controller/simulator.py

```diff
@@ -2,12 +2,22 @@
 This module contains the code to retrieve (hardware-related) data from the EVSE
 (Electric Vehicle Supply Equipment).
 """
+
 import logging
 import time
 from dataclasses import dataclass
 from typing import List, Optional
 
-from aiofile import async_open
+
+from iso15118.shared.utils import install
+
+try:
+    from aiofile import async_open
+except ModuleNotFoundError:
+    install("aiofile")
```

Comment:
Is there a reason not to just move this out of `dev-dependencies` and into `dependencies` in the pyproject.toml file?  We really shouldn't be installing things in one-off ways like this, if we can avoid it at all.

Reply:
Poetry 1.2 is now in beta (https://pypi.org/project/poetry/#history)
<img width="692" alt="image" src="https://user-images.githubusercontent.com/89470921/179545208-4dd47291-83b5-4eef-978a-ad8db127ea0b.png">

Reply:
In the current version of Poetry there is another way of doing this which is by using [extras](https://python-poetry.org/docs/pyproject#extras). So we would put it in the dependencies, but add it as optional for now. Will have a play around and see if this is a solution which could work.